src/main/pages/che-7/end-user-guide/assembly_exporting-a-workspace-with-a-che-factory.adoc

```diff
@@ -0,0 +1,35 @@
+---
+title: "Exporting a workspace with a Che factory"
```

Comment:
Do we really want to promote factory concept in Che 7 ?

Reply:
+1 I don't think we need to include these docs.
We should revamp those and explain how to create factory links using devfile.